pkg/pipelineascode/pipelineascode.go

```diff
@@ -60,7 +58,7 @@ func Run(cs *cli.Clients, k8int cli.KubeInteractionIntf, runinfo *webvcs.RunInfo
 	var err error
 	var maintekton TektonYamlConfig
 
-	var ctx = context.Background()
+	ctx := context.Background()
```

Comment:
Any reason to define `ctx` here even though it is used quite later ?

Reply:
yeah i thought so, what is a good practice in GO ? define at the top or near by where it's used ?

Reply:
Usually, define near where it is used 👼🏼